docs/api/structures/product-discount.md

```diff
@@ -0,0 +1,9 @@
+# ProductDiscount Object
+
+* `identifier` string - A string used to uniquely identify a discount offer for a product.
+* `type` number - The type of discount offer.
+* `price` number - The discount price of the product in the local currency.
+* `priceLocale` string - The locale used to format the discount price of the product.
+* `paymentMode` number - The payment mode for this product discount. `pay_as_you_go: 0`, `pay_up_front: 1`, `free_trial: 2`.
```

Comment:
API-wise, I think exposing this as a string is better than a number. This should be
```suggestion
* `paymentMode` string - The payment mode for this product discount. Can be `payAsYouGo`, `payUpFront`, or `freeTrial`.
```